satpy/tests/reader_tests/test_nucaps.py

```diff
@@ -69,6 +69,7 @@ def get_test_content(self, filename, filename_info, filetype_info):
             ('Solar_Zenith', 'degrees', 'solar_zenith_angle'),
             ('Topography', 'meters', ''),
             ('Land_Fraction', '1', ''),
+            ('Effective_Pressure', 'mb', ''),
```

Comment:
This should be moved to the `for` loop below so the test data is pressure based.